test/plugins/graphql.spec.js

```diff
@@ -2,14 +2,21 @@
 
 const agent = require('./agent')
 
+function findSpan (spans, name, resource, service) {
```

Comment:
This function removes some of the functionality of the previous tests, such as validating the span creation order.

Reply:
Ok 👍